docs/mobile/events.md

```diff
@@ -21,46 +20,57 @@ application lifecycle. This event is meant to be used in conjunction with
 `os.name` [resource semantic convention](/docs/resource/os.md) to identify the
 mobile operating system (e.g. Android, iOS).
 
-### iOS
+Payload attributes MUST be used to describe the state of the application at the
```

Comment:
is "payload attribute" a spec terminology? If not, can we call them "event fields" to avoid confusion ?

Reply:
The `payload` is what we have been calling main payload of an event, the wording here (as I'm just reformatting) pre-dates calling these `payload fields`, I'll update